indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -313,11 +475,17 @@ interface Asn1OctetString<T : Asn1Element> {
 }
 
 
-data class TLV(val tag: UByte, val content: ByteArray) {
+data class TLV(val tag: Asn1Element.Tag, val content: ByteArray) {
```

Comment:
Maybe extract to separate file so this huge file gets a bit smaller?

Reply:
I meant `TLV` at all related methods

Reply:
done